docs/validators/staking-v4.md

```diff
@@ -102,9 +102,12 @@ Staking phase 4 will unfold in three consecutive steps, each corresponding to a
 
 ## Staking v4. Step 1.
 
-In the first step, we will completely **remove the staking queue** and place all nodes in an **auction list**. This
-process will occur automatically at the end of the epoch and requires no interaction from validators. Nodes'
-distribution remains unchanged.
+In the first step, we will completely **remove the staking queue** and **unstake all nodes from the staking queue**. 
+This process will occur automatically at the end of the epoch and requires no interaction from validators.
+Nodes' distribution remains unchanged.
+
+For owners which had **unstaked** nodes, these can be **restaked** using 'RestakeUnstakedNodes' complete details here:
+https://docs.multiversx.com/validators/delegation-manager/#restaking-nodes
```

Comment:
I think for links(here and below) we should use the () [] md links, not simply putting them there

Reply:
[here](https://docs.multiversx.com/validators/delegation-manager/#restaking-nodes)